packages/renderer/src/lib/actions/ContributionActions.spec.ts

```diff
@@ -209,3 +227,23 @@ test('Expect when property to be true multiple args', async () => {
   const item = screen.getByText('dummy-title');
   expect(item).toBeInTheDocument();
 });
+
+test('Expect custom icon on the contributed action', async () => {
```

Comment:
Maybe should also test that default/faPlug is returned if no icon?

Reply:
test added

Reply:
Thanks.